### ISSUE TITLE
Add default currency code or update docs

### ISSUE THREAD

**Issue by Raphtor** [namaggarwal/splitwise#80]
For some reason, I am getting the error

```
{'base': ['Currency code Splitwise does not support the given currency code']}
```
When trying to create an expense without explicitly setting the currency code. Previously this was not an issue, and I suspect something changed in the Splitwise API.

**Comment by Mencucci** [comment]
As of right now, the splitwise documentation at https://dev.splitwise.com/#tag/expenses/paths/~1create_expense/post does **not** mark the `currency_code` field as *required*. Image below of the documentation taken just now
![image](https://user-images.githubusercontent.com/32386969/216574279-ecba8548-d0ea-4cd7-a1f5-4100758d5b36.png)

I am running into the same error and failing to create an expense. Will try to add currency code and update later

**Comment by Mencucci** [comment]
Adding `e.setCurrencyCode("EUR")` in my code made the request go through, now the expense shows up on the Splitwise app on my phone.

+1 on official and SDK's documentation being outdated

**Comment by namaggarwal** [comment]
Fixed in v3.0.0
